SwiftGit2/Repository.swift

```diff
@@ -32,6 +67,47 @@ final public class Repository {
 		let repository = Repository(pointer)
 		return Result.Success(repository)
 	}
+
+	/// Clone the repository from a given URL.
+	///
+	/// remoteURL - The URL of the remote repository
+	/// localURL - The URL to clone the remote repository into
+	///
+	/// Returns a `Result` with a `Repository` or an error.
+	class public func cloneFromURL(remoteURL: NSURL, toWorkingDirectory: NSURL, localClone: Bool = false, bare: Bool = false, checkout: Bool = true,
```

Comment:
This controls whether it copies or fetches the object database. I'm not sure whether we'd want to assume that is always the case when both the URLs are local, happy to infer if you think it would be better though.